interface/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM adage-server/docker-interface-base
+
+# Copy code to server directory
+COPY . .
```

Comment:
This is super minor, but it would be nice if you could substitute the full path in for the second period